src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -80,15 +80,14 @@
      * Private projects configuration, which includes everything, including public settings
      * @param config public project configuration
      */
-    public InternalProjectConfiguration(ProjectConfiguration config ) {
+    public InternalProjectConfiguration(ProjectConfiguration config) {
 
         this.publicConfig = Objects.requireNonNull(config);
 
-        boolean usePrismSW = Boolean.parseBoolean(System.getProperty("prism.sw", "false"));
         boolean skipCompile = Boolean.parseBoolean(System.getProperty("skipcompile", "false"));
         boolean skipSigning = Boolean.parseBoolean(System.getProperty("skipsigning", "false"));
 
-        setUsePrismSW(usePrismSW);
+        setUsePrismSW(config.isUsePrismSW());
```

Comment:
would be good to have a test that checks this: set usePrismSW on public config, then request if from internalconfig